src/components/BlockchainIcon/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import { chainIconsSrcOverrides } from '@/utils/chainIconSrcOverrides'
+import BlockchainIconImg from '@/assets/images/BlockchainIconImg.svg?url'
+import { chainMap } from '@hyperplay/chains'
+
+interface BlockchainIconProps {
+  chainId: string
+  className?: string
+  ipfsBaseUrl?: string
+  /* eslint-disable-next-line */
+  ImageComponent?: any // Allow for a custom image component
```

Comment:
I think if we start using any we shoudl either disable the not `any` rule for good or start typing properly these components. Otherwise there is no porpuse on using TS only for the basic types 😝 

Another point is that, since we are tripling down on tests in general, even though TS won't replace them, it is a important piece of making a high quality code since we will not pass a number or an object here and break the component, `any` will allow that.

This could be something like this:
`ImageComponent?: FC<ImgHTMLAttributes<HTMLImageElement>>`

Reply:
This is not worth our time and isn't very error prone. Mantine uses `component: any` for this as well. Switching the whole project to JS because of 1 `any` is not logical.

Reply:
But that type could work though 
Better than any :P